### PR TITLE
feat: TE7-T1022: Implement DashBoardScreen

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement dashboard screen UI

### CODE REVIEW

File: app/modules/home/useHome.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from '../../redux';
+import { getUsersThunk } from '../../services';
+
+const useHome = () => {
+  const dispatch = useAppDispatch();
+  const {users} = useAppSelector(state => state.home);
```
Comment (amitkajwani-simformsolutions) [inline app/modules/home/useHome.ts]: const { users }

Comment (amitkajwani-simformsolutions) [reply app/modules/home/useHome.ts]: Add space

File: app/redux/home/HomeSlice.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+interface InitialStateType {
+  users: UserSchemaType[] | undefined;
+  page: number;
+  isLoading: boolean;
+  error: string;
+}
+
+const initialState: InitialStateType = {
+  users: undefined,
+  page: 1,
```
Comment (amitkajwani-simformsolutions) [inline app/redux/home/HomeSlice.ts:18]: Don't use static page number in slice

**Review by amitkajwani-simformsolutions — CHANGES_REQUESTED** [review]

File: app/redux/home/HomeSlice.ts (hunk 2 above)
Comment (jigar-s-simform) [reply app/redux/home/HomeSlice.ts:18]: it is initial value of state, any how we should provide any initial number!

File: app/modules/home/useHome.ts (hunk 1 above)
Comment (jigar-s-simform) [reply app/modules/home/useHome.ts]: resolved

File: app/modules/home/HomeScreen.tsx [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+import React from 'react';
+import { FlatList, View } from 'react-native';
+import { UserSchemaType } from '../../services';
+import styles from './HomeStyles';
+import useHome from './useHome';
+import { UserCard } from './user-card';
 
-const HomeScreen: FC = () => <Text>{Strings.bottomNavigationTitles.home}</Text>;
+const HomeScreen = () => {
+  const {users} = useHome();
```
Comment (amitkajwani-simformsolutions) [inline app/modules/home/HomeScreen.tsx]: const { users }

File: app/modules/home/useHome.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from '../../redux';
+import { getUsersThunk } from '../../services';
+
+const useHome = () => {
+  const dispatch = useAppDispatch();
+  const { users } = useAppSelector(state => state.home);
```
Comment (amitkajwani-simformsolutions) [inline app/modules/home/useHome.ts:8]: Add one blank line

**Review by amitkajwani-simformsolutions — CHANGES_REQUESTED** [review]

File: app/modules/home/useHome.ts (hunk 4 above)
Comment (jigar-s-simform) [reply app/modules/home/useHome.ts:8]: resolved

File: app/modules/home/HomeScreen.tsx (hunk 3 above)
Comment (jigar-s-simform) [reply app/modules/home/HomeScreen.tsx]: resolved

**Review by amitkajwani-simformsolutions — APPROVED** [review]

**Pull request merged**